### PR TITLE
Cms/blogs/2023 06 15 registration for opensearchcon 2023 is now open

### LINKED ISSUE #1701: Registration for OpenSearchCon 2023 is now open!
**Describe the blog post your would like to write**
This is a blog post promoting OpenSearchCon 2023 registration and CFP.

**What is the title of the blog post?**
Registration for OpenSearchCon 2023 is now open!  (Full markdown of body below)

[START]
opensearchcon

[OpenSearchCon](https://opensearchcon2023.splashthat.com/) is back! The OpenSearch community will get together again September 27-29 to learn, share, and connect about all things search and analytics. The event is in-person only and there is no fee to attend.

We’re making room for more builders and more content by taking over the Sheraton Grand in Seattle. [Special room rates](https://book.passkey.com/event/50584019/owner/346/home) are available for those who wish to stay at the event hotel. OpenSearchCon 2023 passes are limited, so we encourage everyone to [register today](https://opensearchcon2023.splashthat.com/) in order to claim your spot at this year’s conference. 

The Call for Papers is still open as well but only until the end of the day on June 16. If you have a story you’d like to share, submit your talk NOW on the [Call for Presentations](https://airtable.com/shrVZLtXJ2d1cI6pT) site - time is of the essence!

OpenSearchCon is for builders, with conference tracks designed to explore and learn more about building applications at scale as an open-source contributor.  The current schedule for OpenSearchCon:

*  Sept. 27 - Registration and check-in open (come get your swag!!), exhibit hall is open for exhibitors to set up, and afternoon workshops and round table discussions with partners, contributors and users. There will be an evening cocktail reception as well!
* Sept. 28 - Main conference day with three session rooms/tracks, large exhibit hall, opening/closing keynote, and sessions all day. Food/drinks will be served regularly to keep the community going, and dinner in the main room at the end of the day
* Sept. 29th - Half-day on Friday, focused on buttoning up any missed conversations, and a handful more of user and/or partner round table discussions. 

This year, we will have three session rooms instead of a single main room.  The Search and Analytics tracks will cover a broad range of technology topics, including Open Telemetry, Lucene, Vector Databases, AI/ML, and more. The main stage Community track is where we’ll kick things off with the opening keynote, and will be all about living in the world of open-source software.

First and foremost, this is a community oriented developer meetup - and anyone who wants to become a better builder in this space ought to be there. See you in Seattle!

**[You can learn more and sign up for OpenSearchCon 2023 here.](https://opensearchcon2023.splashthat.com/)**

Keep an eye for this blog for updates on #OpenSearchCon 2023. We’ll see you in Seattle!

[END]

**Who are the authors?**
Me - I have an author profile already.  how do I review and edit my author profile if I want to?

**What is the proposed posting date?**
Patti tells me I have to post it today.

**What front-matter do you want loaded?**
meta_description: "OpenSearchCon 2023 - Coming to the Sheraton Grand Hotel in Seattle September 27-29. Register today!"

excerpt: The OpenSearch Project invites all builders to Seattle this fall, to celebrate the success of the OpenSearch Project and look to the future of search, analytics, and Generative AI. Join us to learn how to build at scale!

authors: 
    - dimwetoth

feature_image: /assets/102261489/13cfc936-f1c7-4e4b-91c5-9927af4a05e2

### PR DESCRIPTION
**Pull request by nateynateynate**
### Description
Publishes blog for opensearchcon 2023
 
### Issues Resolved
#1701

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: assets/media/blog-images/245890710-13cfc936-f1c7-4e4b-91c5-9927af4a05e2.jpg
Comment (krisfreedain) [inline assets/media/blog-images/245890710-13cfc936-f1c7-4e4b-91c5-9927af4a05e2.jpg:1]: please move this file under a '2023-06-15-registration-for-opensearchcon-2023-is-now-open' sub-directory.

File: _posts/2023-06-15-registration-for-opensearchcon-2023-is-now-open.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  opensearchcon registration
+feature_image: /assets/media/blog-images/245890710-13cfc936-f1c7-4e4b-91c5-9927af4a05e2.jpg
+authors:
+  - daryllswager
+layout: post
+title: Registration for OpenSearchCon 2023 is now open!
+categories:
+  - community
+---
+![opensearchcon 2022 stage](/assets/media/blog-images/245890710-13cfc936-f1c7-4e4b-91c5-9927af4a05e2.jpg "opensearchcon crowd")
```
Comment (krisfreedain) [inline _posts/2023-06-15-registration-for-opensearchcon-2023-is-now-open.md:18]: Suggest changing "opensearchcon crowd" to "Laysa Uchoa giving her keynote speech at OpenSearconCon 2022" 

need to have {: .img-fluid}   after the last  )

**Review by krisfreedain — CHANGES_REQUESTED** [review]
a couple changes needed

**Comment by nateynateynate** [comment]
@krisfreedain - Got it. Have another go.

**Review by krisfreedain — APPROVED** [review]
LGTM

**Pull request merged**